### ISSUE TITLE
question about training of TSDAE  

### ISSUE THREAD

**Issue by Gpwner** [UKPLab/sentence-transformers#956]
When I  run the file `train_tsdae_from_file.py` on my 4 V100 sever,I found that only the first GPU been used.
Here is my questions:
1、How to use all my V100 GPUs when running this script?
2、How  can I tell whether the Model converges? Because there is not metric like loss or something

**Comment by Gpwner** [comment]
Here is my training log:

```

2021-05-26 06:25:55 - Start training
2021-05-26 06:31:27 - Save model to output/train_tsdae-_home_roberta-base_-2021-05-26_06-25-44/500
2021-05-26 06:36:58 - Save model to output/train_tsdae-_home_roberta-base_-2021-05-26_06-25-44/1000
Iteration: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1151/1151 [12:39<00:00,  1.51it/s]
2021-05-26 06:42:28 - Save model to output/train_tsdae-_home_roberta-base_-2021-05-26_06-25-44/1500
2021-05-26 06:47:59 - Save model to output/train_tsdae-_home_roberta-base_-2021-05-26_06-25-44/2000
Iteration: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1151/1151 [12:40<00:00,  1.51it/s]
2021-05-26 06:53:30 - Save model to output/train_tsdae-_home_roberta-base_-2021-05-26_06-25-44/2500
2021-05-26 06:59:01 - Save model to output/train_tsdae-_home_roberta-base_-2021-05-26_06-25-44/3000
Iteration: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1151/1151 [12:41<00:00,  1.51it/s]
Epoch: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3/3 [38:02<00:00, 760.89s/it]
2021-05-26 07:04:01 - Save model to output/train_tsdae-_home_roberta-base_-2021-05-26_06-25-44/3453

```

**Comment by nreimers** [comment]
1) Multi-GPU is currently not supported
2) The training loss is not really meaningful to know if the embeddings make sense or not. So it is best to add an evaluator to the fit method that computes a score on some dev dataset.

**Comment by Gpwner** [comment]
> 1. Multi-GPU is currently not supported
> 2. The training loss is not really meaningful to know if the embeddings make sense or not. So it is best to add an evaluator to the fit method that computes a score on some dev dataset.

Why not pass the loss of the original sentence and the reconstruct sentence?

**Comment by nreimers** [comment]
This is currently not implemented. You could look at that loss, but it does sadly not really tell you how well the vector space will be for downstream tasks.

**Comment by Gpwner** [comment]
ok,thanks for  reply.
